code/tables/tex_output/individual_tables/summary_reviewer_char.tex

```diff
@@ -8,14 +8,14 @@
  & \multicolumn{1}{c}{Full sample} & White & Black & Hispanic & Asian
 \\
 \hline\hline\noalign{\smallskip} 
- \textit{Reviewer characteristics} & & & & & \\ \hline \\ Reviewer race & 1.00 & 0.76 & 0.04 & 0.05 & 0.15 \\\\
```

Comment:
@ThiagoSchmidtResende - is this being recreated by code every time we re-run project? or is this just something you're using as a placeholder? 

please don't track individual .tex files! Nothing in any folders like "output" should ever be tracked, because the point of recreating is that you dont have to do this (and it creates merge errors)

Reply:
Yes, its recreated by the code every time. This tex file will be automatically regenerated once we run the code. I committed it so that we can take a look at the tables before running the entire project.